packages/nextjs/components/blockexplorer/TransactionsTable.tsx

```diff
@@ -0,0 +1,100 @@
+import { ethers } from "ethers";
+import { TransactionHash } from "~~/components/blockexplorer/TransactionHash";
+import { Address } from "~~/components/scaffold-eth";
+import { getTargetNetwork } from "~~/utils/scaffold-eth";
+import { TransactionWithFunction, TransactionsTableProps } from "~~/utils/scaffold-eth/";
+
+export const TransactionsTable = ({ blocks, transactionReceipts, isLoading }: TransactionsTableProps) => {
+  const targetNetwork = getTargetNetwork();
+
+  return (
+    <div className="flex justify-center">
+      <table className="table table-zebra w-full shadow-lg">
+        <thead>
+          <tr>
+            <th className="bg-primary">Transaction Hash</th>
+            <th className="bg-primary">Function Called</th>
+            <th className="bg-primary">Block Number</th>
+            <th className="bg-primary">Time Mined</th>
+            <th className="bg-primary">From</th>
+            <th className="bg-primary">To</th>
+            <th className="bg-primary text-end">Value ({targetNetwork.nativeCurrency.symbol})</th>
+          </tr>
+        </thead>
+        {isLoading ? (
+          <tbody>
+            {[...Array(20)].map((_, rowIndex) => (
```

Comment:
If I use `Array(20).map` instead of `[...Array(20)].map`, the skeleton UI does not work.

Reply:
Yeah, `Array(20).map(...)` doesn't do the same, since `Array(20)` on its own creates an array of empties, it's a bit of a js quirk.

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/15879327/ab34a9af-1c38-422e-b4bf-29a49b2c79e8)

I usually do `Array.from({length: 20})`, not sure if there's any benefit/downside to either option.